### PR TITLE
Sign Up 

### PR DESCRIPTION
**Pull request by mohdzakwan99**
Ready

### CODE REVIEW

**Comment by mohdzakwan99** [comment]
ready

File: app/views/shared/_error_messages.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      <%= t (:the_form_contains)%>
+      <%= pluralize @user.errors.count, t(:error) %>.
+    </div>
+    <ul>
+    <% @user.errors.full_messages.each do |msg| %>
```
Comment (huongnt-2545) [inline app/views/shared/_error_messages.html.erb]: wrong indent
```suggestion
      <% @user.errors.full_messages.each do |msg| %>
        <li><%= msg %></li>
      <% end %>
```

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(@user) do |f| %>
+      <% if @user.errors.any? %>
+        <div id="error_explanation">
+          <div class="alert alert-danger">
+            <%= t (:the_form_contains)%>
+            <%= pluralize @user.errors.count, t(:error) %>.
+          </div>
+          <ul>
+          <% @user.errors.full_messages.each do |msg| %>
```
Comment (huongnt-2545) [inline app/views/users/new.html.erb]: wrong indent
```suggestion
            <% @user.errors.full_messages.each do |msg| %>
              <li><%= msg %></li>
            <% end %>
```

File: config/routes.rb [hunk 3]
```diff
@@ -1,7 +1,11 @@
 Rails.application.routes.draw do
-    # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   scope "(:locale)", locale: /#{I18n.available_locales.join("|")}/ do
       root 'static_pages#home', as: "home"
       get 'static_pages/help', as:"help"
+      get "/signup", to: "users#new"
+      post "/signup", to: "users#create"
+      get 'users/show'
```
Comment (huongnt-2545) [inline config/routes.rb]: remove line 7, 8
It's already defined at `resources :users`

File: test/fixtures/users.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (huongnt-2545) [inline test/fixtures/users.yml]: delete this file, it's not useful

**Comment by mohdzakwan99** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**